### PR TITLE
Set up i18n

### PR DESCRIPTION
**Pull request by dgca**
Adds localization to the app!

In order for strings to be picked up for localization, they'll need to be defined using `<FormattedMessage />`. E.g.

```tsx
import { FormattedMessage } from "react-intl";

function Hello() {
  return (
     <Text>
       <FormattedMessage defaultMessage="Hello world!" description="Optional description for translators" />
     </Text>
  );
}
```

For more options, check out the [react-intl docs](https://formatjs.io/docs/react-intl/components#formattedmessage).

---

### Translation process:

- `npm run extract:i18n` will go through the app and extract any strings that need to be localized. This will update the JSON file at `/renderer/locales/en.json`. Note that only the `en.json` file is updated.
- We then need to take that `en.json` file and use it to generated translated JSON files for other supported languages. E.g. we can send it to a translation service which will provide us with `ru.json` and `ch.json`. We then put those files alongside `en.json` in the `/renderer/locales` folder.
- `npm run compile:i18n` will then format the translation files so that the Next.js app can consume them properly.
- `npm run i18n` will run both of the commands above, which should be fine to do in most situations.

### CODE REVIEW

File: renderer/next.config.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ module.exports = {
   images: {
     unoptimized: true,
   },
+  // i18n: {
+  //   locales: ["en-US", "ru-RU"],
+  //   defaultLocale: "en-US",
+  // },
```
Comment (dguenther) [inline renderer/next.config.js:10]: Is this needed for anything?

**Review by dguenther — CHANGES_REQUESTED** [review]
Nice, the process is a bit complex but at the application level it seems like it's not too bad!

File: renderer/next.config.js (hunk 1 above)
Comment (dgca) [reply renderer/next.config.js:10]: Nope!

File: renderer/intl/compiled-locales/en.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "/km2Zu": [
```
[the remark this answers is not in the document]
Comment (dgca) [reply renderer/intl/compiled-locales/en.json]: Just added the compiled folder to gitignore 👍

[the remark this answers is not in the document]
Comment (dguenther) [reply renderer/intl/compiled-locales/en.json]: Nice, I'll take another peek

**Review by dguenther — APPROVED** [review]

**Pull request merged**